rfcs/013-release_deployment_tracking/README.md

```diff
@@ -4,396 +4,211 @@
+![Simple overview](simple_flow.png)
 
-A deployment mechanism to take a container image for a service within a project and run that in a given environment is deliberately not described here.
+More specifically including a high level view of infrastructure includes:
 
-SSM parameters are heavily favoured as they provide a data source for terraform.
+- A build environment to build artifacts, e.g. Travis
+- An artifact store to store things, e.g. ECR
+- A database to keep track of what should be released where e.g. DynamoDB
+- A Deployment environment to run things in, e.g. ECS
 
-## Proposed Solution
+![Infrastructure overvier](high_level_infra.png)
 
-This solution describes a set of steps to track build, release and deployment but is deliberately agnostic of deployment mechanism. In addition this RFC **does not** describe a mechanism for reporting build or deployment status.
+### Glossary
 
-### Build artifacts
+In order that we can talk about the numerous concepts flying about here, we'll define some terms and visualise how they fit together:
 
-> **Build artifact:** A container image encapsulating application code that can be configured by environment variable. The artifact _does not_ contain configuration.
+- **project:** A high level abstraction, consisting of one or more **service set**'s. Practically this might indicate whole product and should be a single git repository, e.g. the "catalogue" project.
 
-When container images intended for release are built they will be added to an ECR repository in AWS.
+- **service set:** A functional grouping of **services** within a project. e.g. all the services for the catalogue pipeline. You can have multiple per project, for example in the catalogue project, you've got pipeline, api and adapters.
 
-#### Build artefacts SSM Parameters
+- **service:** Performs a distinct function within a **project**, practically it may be composed of a few closely related containers. e.g. id_minter, requests API, Front-end content app.
 
-The URI of the container image will then be used to update an SSM parameter with a key:
+- **environment:** Where you deploy your **services** when you want them to run! e.g. staging, production
 
- `/{project_id}/artefacts/{label}/{service_id}`
+- **build environment:** Where you build artifacts to deploy, for example CircleCI, Travis, or your local machine (if tooling allows).
 
-The attributes are described as follows:
-- **project_id**: an identifier for a project.
-- **label:** A label for the set of images released, e.g. latest, stable, v2-branch
+- **build:** The process of creating a **build artifact** for a single **service**
 
- For example:
+- **build artifact:** A deployable _thing_ for a single **service**, practically this is a container image stored in ECR.
 
- `/storage/images/latest/archivist`
+- **release hash:** Metadata uniquely identifying a **build artifact**, practically this will be a git ref. 
```

Comment:
How does a release hash identify a build artefact? Why do we need it?

```suggestion
- **release hash:** Metadata that allows us to work out what version of the code was used to create a given build artifact. For example, the Git commit hash.
```

"hash" is a very Git-specific term. What happened to "release ID"?